common/Util-desktop.cpp

```diff
@@ -160,6 +162,56 @@ std::size_t getTotalSystemMemoryKb()
     return totalMemKb;
 }
 
+size_t getMaxConcurrentTCPConnections(const size_t minimum)
+{
+#ifdef __linux__
```

Comment:
We assume Linux if not mobile everywhere currently - as you can see all around in this module; lets stay consistent.

Reply:
Thought about FreeBSD etc and we do have `#ifdef __linux__` within same compilation unit, so this is not clear.

Reply:
fair enough.